src/pages/browse.tsx

```diff
@@ -0,0 +1,42 @@
+import { Row, Col } from "antd";
+
+import Layout from "../hoc/layout";
+import SideBar from "../components/componentPages/home/sideBar";
```

Comment:
This is `ComponentPages` not the `componentpages`

Reply:
Ticket needs to be created for this task